test/serialbox/core/frontend/gridtools/UnittestSerializer.cpp

```diff
@@ -180,30 +180,30 @@ class GridToolsReadWriteTest : public SerializerUnittestBase {
-        cpu_2d_meta_data(dim1, dim2, 0),                                                     //
-        gpu_2d_meta_data(dim1, dim2, 0),                                                     //
-        cpu_3d_meta_data(dim1, dim2, dim3),                                                  //
-        gpu_3d_meta_data(dim1, dim2, dim3),                                                  //
-        cpu_4d_meta_data(dim1, dim2, dim3, dim4),                                            //
-        gpu_4d_meta_data(dim1, dim2, dim3, dim4),                                            //
-        cpu_2d_real_storage_input(cpu_2d_real_meta_data, "cpu_2d_real_storage_input", -1.0), //
-        gpu_2d_real_storage_input(gpu_2d_real_meta_data, "gpu_2d_real_storage_input", -1.0), //
-        cpu_2d_storage_input(cpu_2d_meta_data, "cpu_2d_storage_input", -1.0),                //
-        gpu_2d_storage_input(gpu_2d_meta_data, "gpu_2d_storage_input", -1.0),                //
-        cpu_3d_storage_input(cpu_3d_meta_data, "cpu_3d_storage_input", -1.0),                //
-        gpu_3d_storage_input(gpu_3d_meta_data, "gpu_3d_storage_input", -1.0),                //
-        cpu_4d_storage_input(cpu_4d_meta_data, "cpu_4d_storage_input", -1.0),                //
-        gpu_4d_storage_input(gpu_4d_meta_data, "gpu_4d_storage_input", -1.0),
-        cpu_2d_real_storage_output(cpu_2d_real_meta_data, "cpu_2d_real_storage_output", -1.0), //
-        gpu_2d_real_storage_output(gpu_2d_real_meta_data, "gpu_2d_real_storage_output", -1.0), //
-        cpu_2d_storage_output(cpu_2d_meta_data, "cpu_2d_storage_output", -1.0),                //
-        gpu_2d_storage_output(gpu_2d_meta_data, "gpu_2d_storage_output", -1.0),                //
-        cpu_3d_storage_output(cpu_3d_meta_data, "cpu_3d_storage_output", -1.0),                //
-        gpu_3d_storage_output(gpu_3d_meta_data, "gpu_3d_storage_output", -1.0),                //
-        cpu_4d_storage_output(cpu_4d_meta_data, "cpu_4d_storage_output", -1.0),                //
-        gpu_4d_storage_output(gpu_4d_meta_data, "gpu_4d_storage_output", -1.0) {
+        cpu_2d_real_meta_data(dim1, dim2),                                             //
+        gpu_2d_real_meta_data(dim1, dim2),                                             //
+        cpu_2d_meta_data(dim1, dim2, 1),                                               //
+        gpu_2d_meta_data(dim1, dim2, 1),                                               //
+        cpu_3d_meta_data(dim1, dim2, dim3),                                            //
+        gpu_3d_meta_data(dim1, dim2, dim3),                                            //
+        cpu_4d_meta_data(dim1, dim2, dim3, dim4),                                      //
+        gpu_4d_meta_data(dim1, dim2, dim3, dim4),                                      //
+        cpu_2d_real_storage_input(cpu_2d_real_meta_data, "cpu_2d_real_storage_input"), //

```

Comment:
no init needed anymore?

Reply:
Using lambdas to initialize would simplify the thing